showkase-browser-testing/src/androidTest/java/com/airbnb/android/showkase_browser_testing/ShowkaseBrowserTest.kt

```diff
@@ -544,4 +648,34 @@ class ShowcaseBrowserTest {
             verifyButtonWithTagIsDisplayedAndEnabled("SearchIcon")
         }
     }
+
+    @Test
+    fun stacked_preview_show_up_in_browser() {
+        // Stacked previews are only supported from ksp, so this is to bypass kapt on CI
+        if (System.getProperty("useKsp") == "true") {
```

Comment:
Wouldn't you need to use `if(BuildConfig.IS_RUNNING_KSP)` here instead of System.getProperty?

Reply:
Oh no! Yes you are totally correct! I guess they where just not running on the new test then 🤔 Changed it up in [f2003ad](https://github.com/airbnb/Showkase/pull/259/commits/f2003ad60499ea65cd6e1ed304de93392f6770f6) Thanks for catching that! :)